source/Api.cpp

```diff
@@ -44,6 +45,7 @@ namespace Aws
         {
             // sets up the StlAllocator for use.
             g_allocator = allocator;
+            aws_checksums_library_init(allocator);
```

Comment:
s3 depends on checksum, we should not need this?

Reply:
🤷‍♀️it's harmless to multiple-init, since the tree of dependencies isn't 100% clear, and aws-crt-cpp **does** directly call into checksums

but really, I need to get a release out today, and latest-submodules brings this fix, and I don't want to wait while I update several other libraries, and thought if I did this here now I could resolve that issue on the C++ SDK

Reply:
nevermind. rolling back. Dmitriy is also working on related fixes